gravitee-apim-portal-webui-next/src/app/registration/registration.component.html

```diff
@@ -0,0 +1,118 @@
+<!--
+
+    Copyright (C) 2024 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (!submitted) {
+  <form
+    [formGroup]="registrationForm"
+    (ngSubmit)="register()"
+    class="registration__form__container"
```

Comment:
Related to https://github.com/gravitee-io/gravitee-api-management/pull/14707/changes#r2640376291

```suggestion
    class="registration__form__container content"